src/server/scripts/Outland/CoilfangReservoir/SteamVault/boss_hydromancer_thespia.cpp

```diff
@@ -15,172 +15,123 @@
+    boss_hydromancer_thespia(Creature* creature) : BossAI(creature, DATA_HYDROMANCER_THESPIA) { }
+
+    void JustEngagedWith(Unit* who) override
+    {
+        Talk(SAY_AGGRO);
+        BossAI::JustEngagedWith(who);
+
+        events.ScheduleEvent(EVENT_LIGHTNING_CLOUD, 10s, 15s);
+        events.ScheduleEvent(EVENT_LUNG_BURST, 7s, 12s);
+        events.ScheduleEvent(EVENT_ENVELOPING_WINDS, 10s, 15s);
+    }
+
+    void KilledUnit(Unit* who) override
+    {
+        if (who->GetTypeId() == TYPEID_PLAYER)
+            Talk(SAY_SLAY);
+    }
+
+    void JustDied(Unit* /*killer*/) override
+    {
+        Talk(SAY_DEATH);
+        _JustDied();
+    }
+
+    void ExecuteEvent(uint32 eventId) override
+    {
+        switch (eventId)
         {
-            boss_thespiaAI(Creature* creature) : BossAI(creature, DATA_HYDROMANCER_THESPIA) { }
-
-            void Reset() override
-            {
-                _Reset();
-            }
-
-            void JustDied(Unit* /*killer*/) override
-            {
-                Talk(SAY_DEAD);
-                _JustDied();
-            }
-
-            void KilledUnit(Unit* who) override
-            {
-                if (who->GetTypeId() == TYPEID_PLAYER)
-                    Talk(SAY_SLAY);
-            }
-
-            void JustEngagedWith(Unit* who) override
-            {
-                Talk(SAY_AGGRO);
-                BossAI::JustEngagedWith(who);
-
-                events.ScheduleEvent(EVENT_LIGHTNING_CLOUD, 15s);
-                events.ScheduleEvent(EVENT_LUNG_BURST, 7s);
-                events.ScheduleEvent(EVENT_ENVELOPING_WINDS, 9s);
-            }
-
-            void ExecuteEvent(uint32 eventId) override
-            {
-                switch (eventId)
-                {
-                    case EVENT_LIGHTNING_CLOUD:
-                        if (Unit* target = SelectTarget(SelectTargetMethod::Random, 0, 30.0f, true))
-                            DoCast(target, SPELL_LIGHTNING_CLOUD);
-                        // cast twice in Heroic mode
-                        if (IsHeroic())
-                            if (Unit* target = SelectTarget(SelectTargetMethod::Random, 0, 30.0f, true))
-                                DoCast(target, SPELL_LIGHTNING_CLOUD);
-
-                        events.ScheduleEvent(EVENT_LIGHTNING_CLOUD, 15s, 25s);
-                        break;
-                    case EVENT_LUNG_BURST:
-                        if (Unit* target = SelectTarget(SelectTargetMethod::Random, 0, 40.0f, true))
-                            DoCast(target, SPELL_LUNG_BURST);
-                        events.ScheduleEvent(EVENT_LUNG_BURST, 7s, 12s);
-                        break;
-                    case EVENT_ENVELOPING_WINDS:
-                        if (Unit* target = SelectTarget(SelectTargetMethod::Random, 0, 35.0f, true))
-                            DoCast(target, SPELL_ENVELOPING_WINDS);
-                        // cast twice in Heroic mode
-                        if (IsHeroic())
```

Comment:
intended to remove the heroic only part?

Reply:
Yes, not confirmed